pcapgo/capture_test.go

```diff
@@ -0,0 +1,34 @@
+package pcapgo_test
+
+import (
+	"log"
+	"os"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcapgo"
+)
+
+func Example_captureEthernet() {
```

Comment:
nit: CaptureEthernet

Reply:
This is a load-bearing lowercase “c”: if I make it Example_CaptureEthernet, godoc won’t show the example.

Reply:
lolwut